packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -169,11 +183,11 @@ export class VaAccordion {
             onClick={() => this.expandCollapseAll(!this.expanded)}
             aria-label={
               this.expanded
-                ? 'Collapse all accordions'
-                : 'Expand all accordions'
+                ? i18next.t('collapse-all-aria-label')
+                : i18next.t('expand-all-aria-label')
             }
           >
-            {this.expanded ? 'Collapse all -' : 'Expand all +'}
+            {this.expanded ? `${i18next.t('collapse-all')} -` : `${i18next.t('expand-all')} +`}
```

Comment:
We don't need these changes in here - I'm guessing these were an accident?

Reply:
Very strange, I didn't make these changes in my commit. It might be some weird merging issue? I just tried to rebase off of your branch and force-push, and it still looks the same 😞

Reply:
Once, your PR is merged, I'll clean up this commit history

Reply:
Can you try rebasing from master? That might fix this, and I also recently merged a PR in so that we don't have to fiddle with storybook aliases/imports in these PRs.